src/libraries/System.IO.Hashing/tests/Crc32Tests.cs

```diff
@@ -93,6 +93,7 @@ public void InstanceAppendAllocateAndReset(TestCase testCase)
             InstanceAppendAllocateAndResetDriver(testCase);
         }
 
+        [SkipOnCoreClr("https://github.com/dotnet/runtime/issues/54007", RuntimeConfiguration.Checked)]
```

Comment:
FYI you can go more granular and disable for stress runs but not regular checked runs.

Reply:
I thought about that, but it's any run with optimization (and Checked), so not just stress. Probably should disable also for Debug, but that's rarely used.